src/wallets/lightwallet/LightWallet.ts

```diff
@@ -4,14 +4,18 @@ import { keystore, signing } from 'eth-lightwallet';
 import { Store } from '../../Store';
 import { LightWalletUtils } from './LightWalletUtils';
 
+// Type alias is required here to resolve this issue:
+// https://stackoverflow.com/questions/48640828/class-is-referenced-directly-or-indirectly-in-its-own-type-annotation
```

Comment:
Same fix, but isn't this necessary just because `signing` is a namespace & namespaces can't be used as types?

Reply:
I found it odd, too.

The issue we are coming up against is explained here: https://stackoverflow.com/questions/48640828/class-is-referenced-directly-or-indirectly-in-its-own-type-annotation